package.json

```diff
@@ -19,5 +28,10 @@
   "license": "MIT",
   "contributors": [
     "Francois-Guillaume Ribreau <npm@fgribreau.com>"
-  ]
+  ],
+  "scripts": {
+    "compile": "make build",
```

Comment:
I just forgotten to remove the compiled files from git. They were useful for us before we published our fork with another name. I wanted to do something like described in http://jamesknelson.com/writing-npm-packages-with-es6-using-the-babel-6-cli/.